pkg/kubelet/volume/cache/actual_state_of_world.go

```diff
@@ -127,12 +132,15 @@ type ActualStateOfWorld interface {
 	// current actual state of the world.
 	GetMountedVolumesForPod(podName volumetypes.UniquePodName) []MountedVolume
 
-	// GetAttachedVolumes generates and returns a list of all attached volumes.
-	GetAttachedVolumes() []AttachedVolume
+	// GetGloballyMountedVolumes generates and returns a list of all attached
+	// volumes that are globally mounted. This list can be used to determine
```

Comment:
the "globally" qualifier makes it sound like it is across nodes - can you clarify?